eZ/Publish/Core/FieldType/Url/UrlStorage.php

```diff
@@ -62,6 +62,12 @@ public function storeFieldData(VersionInfo $versionInfo, Field $field, array $co
 
         $this->gateway->linkUrl($urlId, $field->id, $versionInfo->versionNo);
 
+        $this->gateway->unlinkUrl(
+            $field->id,
+            $versionInfo->versionNo,
+            [$urlId]
+        );
```

Comment:
This looks weird together. How does it work?

Reply:
This work like here: https://github.com/ezsystems/ezpublish-kernel/pull/2957
This is the same function to unlink old links from URL's tables

Reply:
well, it is not to unlink old links but rather to unlink all but new ones, right?
@alongosz 
@mateuszdebinski

Reply:
The last parameter of the function indicates which urls should not be removed